crates/bevy_gizmos/src/lib.rs

```diff
@@ -90,6 +90,10 @@ pub struct GizmoPlugin;
 
 impl Plugin for GizmoPlugin {
     fn build(&self, app: &mut bevy_app::App) {
+        // Gizmos cannot work without either a 3D or 2D renderer.
+        #[cfg(all(not(feature = "bevy_pbr"), not(feature = "bevy_sprite")))]
+        bevy_utils::tracing::error!("bevy_gizmos requires either bevy_pbr or bevy_sprite. Please enable one as an optional dependency.");
```

Comment:
Why a "optional" dependency? That wording is slightly weird.

Reply:
I think because they are optional dependencies in `bevy_gizmos`'s `Cargo.toml`. But I do agree that the wording is weird. How about:

```
bevy_gizmos requires either bevy_pbr or bevy_sprite. Please enable one in your Cargo.toml file.
```

Reply:
Nah I think this is would be better:
```
bevy_gizmos requires either the bevy_pbr or the bevy_sprite feature. Please enable one.
```
(Features don't necessarily have to be enabled in `Cargo.toml`)

Reply:
Also it appears Bevy is using a mix of both `bevy_utils::tracing` and `bevy_log` for accessing logs. Interesting. Might wanna fix that sometime.

Reply:
Added in 96a61c04a6d358d27c1d29272094175dcc89aa97. `Please enable one.` seems really short, but it does get straight to the point.

I worry what will happen if a beginner starts using this error message and is unsure how to enable optional dependencies. Any ideas?

Reply:
> I worry what will happen if a beginner starts using this error message and is unsure how to enable optional dependencies. Any ideas?

That won't really happen unless they disable `default-features` in which case they probably know what they are doing.

Reply:
Sounds good :)